content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -129,24 +147,30 @@ Recurring Prices tab
 --------------------
 
 Recurring prices are specifically used with :doc:`subscription products
-<../../../subscriptions/products>`.
+<../../../subscriptions/products>`. When configuring price rules in this tab, keep in mind that
+they will only apply to subscription products, or products with recurring prices enabled.
+
+In the :guilabel:`Recurring Prices` tab, pricelists are configured with the same options as in the
+:guilabel:`Price Rules` tab, with additional columns for :guilabel:`Product Variants` add
+::guilabel:`Recurring Plan`.
 
-Under the :guilabel:`Recurring Prices` tab, the same functionality of the :guilabel:`Price Rules`
-tab is present. The only difference being that a recurring time period can be applied in the
-:guilabel:`Recurring Plan` column.
+:guilabel:`Product Variants` are configured under products that have one or more values, such as
+color, size, etc. Once a product has been selected under the :guilabel:`Products Tab`, if applicable
+, select the desired product variants to be included in the price rule.
```

Comment:
Please only break a line where a ` ` space exists, otherwise a space is added due to the line break.



```suggestion
color, size, etc. Once a product has been selected under the :guilabel:`Products Tab`, if
applicable, select the desired product variants to be included in the price rule.
```
![Screenshot 2025-01-30 at 5 10 37 PM](https://github.com/user-attachments/assets/3d390c62-ebed-42ca-b12c-ec8f275e5cef)